language-sql.md

```diff
@@ -3,11 +3,33 @@
 ### Minimum Requirements
 
 - MySQL keywords in uppercase
-- Column names in lowercase
+    E.g.: 
+    ```
+    UPDATE `consumer` SET `PASSWORD` = MD5(`PASSWORD`);
+    ```
+
+- Database table names must be plural. 
+    E.g.: 
+    `users`, `subscriptions`
+    
+    for Linking tables :
+    `users_subscriptions`
+
+- Column names in lowercase and must be singular. Avoid using MySQL reserved words such as order, status, etc.
```

Comment:
Based on https://tools.ietf.org/html/rfc2119 (thanks @johndavedecano):

Column names **MUST** be lowercase and **MUST** be singular. They **MUST NOT** be MySQL reserved words: order, status, etc.